C++ Lib and Linux style guide.md

```diff
@@ -804,3 +804,45 @@ Only use spaces (not tabs).
 ## `UseCRLF: false`
 
 This project is primarily for Linux platforms, so we prefer to have LF line endings. We don't use Windows-style CRLF line endings.
+
+## `EmptyLineAfterAccessModifier: Leave`
+
+Allows empty lines after a data structure's access modifiers.
+
+```cpp
+class MyClass {
+    
+public:
+
+    MyClass();
+
+private:
+    _some_private_function();
+};
+```
+
+## `IndentAccessModifiers: false`
+
+Do not indent data structures' access modifiers. They should be the same indentation as the structure's definition.
+
+## `ReferenceAlignment: Right`
+
+A reference operator (`&`) should be aligned to the right (immediately preceding the object name).
```

Comment:
Please see the [possible settings](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#referencealignment) for this option.

I'm going with `Right` because it reads better (IMO) and is similar to how the operator is used in rust (for similar behavior).

Reply:
I'm changing my mind. If I set this value to `Pointer`, then I there will be less format changes to the source code when upgrading to v14 compatible .clang-format config files.